tethysext/atcore/services/geoserver_api.py

```diff
@@ -311,7 +311,9 @@ def create_postgis_store(self, workspace, name, db_host, db_port, db_name, db_us
         )
 
         # Return with error if this doesn't work
-        if response.status_code != 201:
```

Comment:
What would it take to remove this file entirely and replace it's usage with the equivalent methods in the tethys_dataset_services API?

Reply:
Four hours at most. I checked and GeoServerAPI is only used in the init command and the base spatial manager. All uses call the class so it would be easy to just switch the import to come from tds. The rest would be cleaning up, including a bunch of tests that could be removed.

Reply:
Do it.

Reply:
Yes, agwa would require some refactoring as well. It uses atcore base spatial manager class and calls various gs_api functions using "self.gs_api"

These are the functions agwa uses: get_layer_extent, create_layer, create_style, delete_style. The main change in these is that before we took workspace and name parameters and now we take a store_id or layer_id that is a combination of the two (e.g. workspace:name), so the refactoring would not be too complicated.